packages/core/src/oidc/init.ts

```diff
@@ -28,7 +28,7 @@ export default async function initOidc(app: Koa): Promise<Provider> {
     path: '/',
     signed: true,
   } as const);
-  const oidc = new Provider(issuer, {
+  const oidc = new Provider(issuer.toString(), {
```

Comment:
<img width="644" alt="image" src="https://user-images.githubusercontent.com/12833674/177294983-934bd8d9-47e0-4635-a5e4-4c58d55ca9e2.png">

Seems `issuer` is string type already

Reply:
yes will update